tests/test_admin/test_utilities.py

```diff
@@ -100,7 +101,7 @@ def test_parse_csv_check_with_wrong_encoding(self):
         with MakeCsvStreamContextManager(header, contents, 'utf-16') as stream:
             expected_error_message = ValidationMessages.INVALID_ENCODING
 
-            with raises(ValidationError, message=expected_error_message):
+            with raises(ValidationError, match=expected_error_message):
```

Comment:
Why is it different from line 91 in the same file above? You can also call `pylint.fail(error_message)` at the end of the function.